src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Identity/LogoutForm.razor

```diff
@@ -0,0 +1,34 @@
+﻿@using Microsoft.AspNetCore.Identity
+@using BlazorWeb_CSharp.Data
+@using BlazorWeb_CSharp.Identity
+
+@inject SignInManager<ApplicationUser> SignInManager
+@inject NavigationManager NavigationManager
+@inject IdentityRedirectManager RedirectManager
+
+<form @onsubmit="OnSubmitAsync" @formname="logout" method="post" @attributes="AdditionalAttributes">
```

Comment:
I must be missing something here. How can this form ever get submitted? There's no UI inside it, so no submit button, nor any textbox you could press 'enter' on.

Reply:
I know it's a bit weird, but it's the "Logout" `<a>` element in the `ManageNavMenu` that submits the form via JavaScript. We could have had a `<button>` here instead, but it was a massive pain for me to try to get the styling to match that of the `<NavLink>`s in the menu.